app/views/course/leaderboards/show.html.slim

```diff
@@ -1 +1,43 @@
-p To be implemented later
+= page_header t('course.leaderboard.index.header')
```

Comment:
Okay. I'll stick to convention first, then do the PR later on. There will be some disjoint translation keys cause I used singular form (`leaderboard`) for the sidebar, so I'll add a Todo.